extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -89,33 +90,62 @@ describe('A4A loader', () => {
         });
       });
 
-      it('adds network-specific child for registered, A4A type', () => {
+      it('upgrades to registered, A4A type network-specific element', () => {
         return createIframePromise().then(fixture => {
           const doc = fixture.doc;
+          const extensionsMock = sandbox.mock(extensionsFor(fixture.win));
           a4aRegistry['zort'] = function() {
             return true;
           };
           const element = doc.createElement(tag);
           element.setAttribute('type', 'zort');
-          element.setAttribute('width', '300');
-          element.setAttribute('height', '200');
           doc.body.appendChild(element);
+          const zortInstance = {};
+          const zortConstructor = function() { return zortInstance; };
+          extensionsMock.expects('loadElementClass')
+            .withExactArgs('amp-ad-network-zort-impl')
```

Comment:
please keep indentation of 4 spaces for line breaks. 
Applies to the whole file.